packages/components/src/text-input/test/index.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
We should update these tests to only minimally use `toMatchSnapshot`. Snapshot diffs and snapshot style diffs are probably more helpful.

Reply:
Could you expand on what you mean here? By "minimally use" do we mean that we can remove some or most of the tests?